modules/utxo-lib/package.json

```diff
@@ -18,15 +18,21 @@
     "javascript"
   ],
   "scripts": {
+    "prepare": "npm run build",
```

Comment:
Why have this alias for the `build` script?

Reply:
this is a proper `npm` hook https://docs.npmjs.com/cli/v7/using-npm/scripts#life-cycle-scripts

without it `yarn install` at the toplevel does not work correctly and uses a stale `dist/`